# -*- coding: utf-8 -*-
"""Untitled20.ipynb

Automatically generated by Colaboratory.

Original file is located at
    https://colab.research.google.com/drive/1Uk6oR-uDlgFBeRAeQ5iw5Ekx8Ir6zPDb
"""

import sqlite3

connection = sqlite3.connect('teatchers.db')
cursor = connection.cursor()
query = """CREATE TABLE Students (
Student_Id INTEGER NOT NULL,
Student_Name TEXT NOT NULL,
School_Id INTEGER NOT NULL PRIMARY KEY);
"""
cursor.execute(query)
connection.commit()
connection.close()

import sqlite3

connection = sqlite3.connect('teatchers.db')
cursor = connection.cursor()
query = """INSERT INTO Students (Student_Id, Student_Name, School_Id)
VALUES
(201, 'Иван', 1),
(202, 'Петр', 2),
(203, 'Анастасия', 3),
(204, 'Игорь', 4);"""
cursor.execute(query)
connection.commit()
connection.close()



import sqlite3
def close_connection(connection):
   if connection:
    connection.close()

def get_stud(student_id):
  try:
    connection = sqlite3.connect("teatchers.db")
    cursor = connection.cursor()
    query = """SELECT * FROM School JOIN Students ON School.School_id = Students.School_id WHERE Students.Student_id = ?"""
    cursor.execute(query,(student_id,))
    records = cursor.fetchall()
    print ('Данные по студенту:')

    for row in records:
      print('ID студентуа:', row[3])
      print('Имя студента:', row[4])
      print('ID школы:', row[0])
      print('Название школы:', row[1])


    close_connection(connection)


  except (Exception, sqlite3.Error) as error:
    print('Ошибка в получении данных', error)


number = input('Введите ID студента: ')
print(get_stud(number))